src/DynamoCore/BuiltInAndOperators/BuiltIn.xml

```diff
@@ -70,6 +70,7 @@
       <summary>Returns the items from the list that generate the minimum values for the function supplied as the key projector</summary>
       <param name="list">list of values</param>
       <param name="keyProjector">function applied to the list items</param>
+      <returns name="minItem">minimum item in list using keyProjector</returns>
```

Comment:
Should this be `minimumItem`? or does that get too long. Just seems like we are being more specific with everything else so might make sense to do the same here

Reply:
minItem is probs fine, but worth considering with all the other changes we are doing.

Reply:
Im going to change it to minimumItem, although its a bit longer I think its better to keep consistency. 
b8dc25c